src/pages/experiments/[experimentId]/plots-and-tables/dot-plot/index.jsx

```diff
@@ -212,6 +215,20 @@ const DotPlotPage = (props) => {
     }
   }, [highestDispersionGenes, config, genesFetching]);
 
+  const getCSVData = () => {
+    const newData = plotData?.map(({
+      avgExpression, cellsPercentage, geneName, cellSets: clusterName,
+    }) => ({
+      Gene: geneName,
+      Cluster: clusterName,
+      'Average expression': avgExpression,
+      'Proportion of cells expressing gene': cellsPercentage,
```

Comment:
Would it be better for the header to be "Fraction" of cells instead of "Proportion" of cells?